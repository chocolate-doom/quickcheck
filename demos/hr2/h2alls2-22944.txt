PWAD:		hr2final.wad
PWAD Name:	Hell Revealed II
Map:		Movie 01-32
Skill:		2
Category:	HNTR-Max
Exe:		PrBoom+ v2.5.1.5
Time:		2:29:44

-complevel 2

Author: tchkb



MAP01: 1:11
MAP02: 1:08
MAP03: 2:10
MAP04: 2:10
MAP05: 1:11
MAP06: 1:47
MAP07: 2:44
MAP08: 1:56
MAP09: 4:08
MAP10: 2:16
MAP11: 2:09
MAP12: 3:26
MAP13: 3:30
MAP14: 3:56
MAP15: 9:13
MAP31: 7:24
MAP32: 8:03
MAP16: 5:28
MAP17: 3:35
MAP18: 5:08
MAP19: 7:40
MAP20: 3:01
MAP21: 3:25
MAP22: 5:44
MAP23: 8:16
MAP24: 8:20
MAP25: 4:43
MAP26: 5:33
MAP27: 5:24
MAP28: 16:36
MAP29: 7:22
MAP30: 1:07