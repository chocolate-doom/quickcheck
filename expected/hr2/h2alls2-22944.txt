===========================================
MAP01
===========================================

Time: 1:11 (par: 0:30)

Player 1 (Green):
	Kills: 28 / 28 (100%)
	Items: 1 / 35 (2%)
	Secrets: 2 / 2 (100%)

===========================================
MAP02
===========================================

Time: 1:08 (par: 1:30)

Player 1 (Green):
	Kills: 47 / 47 (100%)
	Items: 1 / 2 (50%)
	Secrets: 3 / 3 (100%)

===========================================
MAP03
===========================================

Time: 2:10 (par: 2:00)

Player 1 (Green):
	Kills: 70 / 70 (100%)
	Items: 2 / 3 (66%)
	Secrets: 2 / 2 (100%)

===========================================
MAP04
===========================================

Time: 2:10 (par: 2:00)

Player 1 (Green):
	Kills: 60 / 56 (107%)
	Items: 3 / 3 (100%)
	Secrets: 2 / 2 (100%)

===========================================
MAP05
===========================================

Time: 1:11 (par: 1:30)

Player 1 (Green):
	Kills: 43 / 43 (100%)
	Items: 5 / 10 (50%)
	Secrets: 2 / 2 (100%)

===========================================
MAP06
===========================================

Time: 1:47 (par: 2:30)

Player 1 (Green):
	Kills: 29 / 29 (100%)
	Items: 2 / 2 (100%)
	Secrets: 3 / 3 (100%)

===========================================
MAP07
===========================================

Time: 2:44 (par: 2:00)

Player 1 (Green):
	Kills: 138 / 138 (100%)
	Items: 2 / 3 (66%)
	Secrets: 2 / 2 (100%)

===========================================
MAP08
===========================================

Time: 1:56 (par: 2:00)

Player 1 (Green):
	Kills: 116 / 116 (100%)
	Items: 4 / 4 (100%)
	Secrets: 1 / 1 (100%)

===========================================
MAP09
===========================================

Time: 4:08 (par: 4:30)

Player 1 (Green):
	Kills: 196 / 196 (100%)
	Items: 5 / 6 (83%)
	Secrets: 1 / 1 (100%)

===========================================
MAP10
===========================================

Time: 2:16 (par: 1:30)

Player 1 (Green):
	Kills: 55 / 55 (100%)
	Items: 4 / 4 (100%)
	Secrets: 1 / 1 (100%)

===========================================
MAP11
===========================================

Time: 2:09 (par: 3:30)

Player 1 (Green):
	Kills: 58 / 57 (101%)
	Items: 6 / 6 (100%)
	Secrets: 1 / 1 (100%)

===========================================
MAP12
===========================================

Time: 3:26 (par: 2:30)

Player 1 (Green):
	Kills: 102 / 97 (105%)
	Items: 2 / 3 (66%)
	Secrets: 2 / 2 (100%)

===========================================
MAP13
===========================================

Time: 3:30 (par: 2:30)

Player 1 (Green):
	Kills: 151 / 151 (100%)
	Items: 4 / 5 (80%)
	Secrets: 1 / 1 (100%)

===========================================
MAP14
===========================================

Time: 3:56 (par: 2:30)

Player 1 (Green):
	Kills: 106 / 106 (100%)
	Items: 5 / 5 (100%)
	Secrets: 1 / 1 (100%)

===========================================
MAP15
===========================================

Time: 9:13 (par: 3:30)

Player 1 (Green):
	Kills: 370 / 368 (-77%)
	Items: 6 / 6 (100%)
	Secrets: 3 / 3 (100%)

===========================================
MAP31
===========================================

Time: 7:24 (par: 2:00)

Player 1 (Green):
	Kills: 335 / 331 (-96%)
	Items: 6 / 8 (75%)
	Secrets: 2 / 2 (100%)

===========================================
MAP32
===========================================

Time: 8:03 (par: 0:30)

Player 1 (Green):
	Kills: 832 / 821 (21%)
	Items: 10 / 18 (55%)
	Secrets: 1 / 1 (100%)

===========================================
MAP16
===========================================

Time: 5:28 (par: 2:30)

Player 1 (Green):
	Kills: 213 / 212 (100%)
	Items: 3 / 5 (60%)
	Secrets: 7 / 7 (100%)

===========================================
MAP17
===========================================

Time: 3:35 (par: 7:00)

Player 1 (Green):
	Kills: 118 / 118 (100%)
	Items: 3 / 3 (100%)
	Secrets: 1 / 1 (100%)

===========================================
MAP18
===========================================

Time: 5:08 (par: 2:30)

Player 1 (Green):
	Kills: 213 / 212 (100%)
	Items: 43 / 45 (95%)
	Secrets: 1 / 1 (100%)

===========================================
MAP19
===========================================

Time: 7:40 (par: 3:30)

Player 1 (Green):
	Kills: 263 / 263 (100%)
	Items: 12 / 12 (100%)
	Secrets: 3 / 3 (100%)

===========================================
MAP20
===========================================

Time: 3:01 (par: 2:30)

Player 1 (Green):
	Kills: 140 / 138 (101%)
	Items: 3 / 5 (60%)
	Secrets: 1 / 1 (100%)

===========================================
MAP21
===========================================

Time: 3:25 (par: 4:00)

Player 1 (Green):
	Kills: 87 / 87 (100%)
	Items: 4 / 4 (100%)
	Secrets: 1 / 1 (100%)

===========================================
MAP22
===========================================

Time: 5:44 (par: 2:30)

Player 1 (Green):
	Kills: 193 / 193 (100%)
	Items: 3 / 6 (50%)
	Secrets: 2 / 2 (100%)

===========================================
MAP23
===========================================

Time: 8:16 (par: 3:00)

Player 1 (Green):
	Kills: 311 / 311 (100%)
	Items: 66 / 70 (94%)
	Secrets: 2 / 2 (100%)

===========================================
MAP24
===========================================

Time: 8:20 (par: 2:30)

Player 1 (Green):
	Kills: 226 / 226 (100%)
	Items: 11 / 12 (91%)
	Secrets: 5 / 5 (100%)

===========================================
MAP25
===========================================

Time: 4:43 (par: 2:30)

Player 1 (Green):
	Kills: 290 / 282 (102%)
	Items: 4 / 5 (80%)
	Secrets: 1 / 1 (100%)

===========================================
MAP26
===========================================

Time: 5:33 (par: 5:00)

Player 1 (Green):
	Kills: 192 / 192 (100%)
	Items: 4 / 5 (80%)
	Secrets: 3 / 3 (100%)

===========================================
MAP27
===========================================

Time: 5:24 (par: 5:30)

Player 1 (Green):
	Kills: 321 / 318 (100%)
	Items: 8 / 9 (88%)
	Secrets: 4 / 4 (100%)

===========================================
MAP28
===========================================

Time: 16:36 (par: 7:00)

Player 1 (Green):
	Kills: 371 / 304 (-93%)
	Items: 4 / 4 (100%)
	Secrets: 6 / 6 (100%)

===========================================
MAP29
===========================================

Time: 7:22 (par: 5:00)

Player 1 (Green):
	Kills: 570 / 569 (-15%)
	Items: 7 / 7 (100%)
	Secrets: 5 / 5 (100%)

===========================================
MAP30
===========================================

Time: 1:07 (par: 3:00)

Player 1 (Green):
	Kills: 40 / 37 (108%)
	Items: 3 / 3 (100%)
	Secrets: 1 / 1 (100%)

